### ISSUE TITLE
Asks for `eslint-plugin-eslint-plugin-react`, which does not exist

### ISSUE THREAD

**Issue by sirbrillig** [AtomLinter/linter-eslint#112]
![screen shot 2015-06-18 at 4 42 19 pm](https://cloud.githubusercontent.com/assets/2036909/8241794/62e7a526-15d9-11e5-81b9-a5e34c5c1409.png)

"`npm install eslint-plugin-eslint-plugin-react` in your project (linter-eslint)"

FWIW I have  `eslint-plugin-react` installed both globally and locally to the project.

**Comment by sirbrillig** [comment]
I managed to make this disappear by removing the following plugin from my `.eslintrc` (I wasn't actively using React at the moment):

```json
	"plugins": [ "eslint-plugin-react" ],
```

However, I believe that this is still an error. The [ESLint docs](http://eslint.org/docs/user-guide/configuring.html#configuring-plugins) read

> The `eslint-plugin-` prefix can be omitted from the plugin name.

...but from what I can tell it seems to be perfectly acceptable to keep the prefix, and it doesn't produce an error on the command-line (nor in vim or Sublime).

**Comment by AsaAyers** [comment]
Yes, because this is running inside of Atom and not your command-line Node we have to [load the plugins manually](https://github.com/AtomLinter/linter-eslint/blob/master/lib/linter-eslint.coffee#L99-L111). 

You can also solve this immediately with

```json
	"plugins": [ "react" ],
```

**Comment by sirbrillig** [comment]
True. I'm just thinking that because the prefix is optionally there, perhaps the code should check to see if there is a prefix on the `pluginName` before prepending it?

**Comment by AsaAyers** [comment]
yup, I'll have a PR up for it soon. trying it out locally

**Comment by zertosh** [comment]
Somewhat related. Namespaced plugins don't work either. eslint's own solution is detailed here https://github.com/eslint/eslint/issues/2360

**Comment by AsaAyers** [comment]
Thanks, I have updated my PR to account for namspaced plugins.

**Comment by iam4x** [comment]
Thank's @AsaAyers for the fix, merging and releasing new version!

**Comment by zertosh** [comment]
@AsaAyers some change in 1.x broke namespaced plugins

**Comment by AsaAyers** [comment]
I think your config is wrong it should either be "react" or
"eslint-plugin-react"

**Comment by zertosh** [comment]
@AsaAyers ah sorry, I'm not talking about `eslint-plugin-react`, I created https://github.com/AtomLinter/linter-eslint/issues/130 with more details
